homeassistant/components/group/strings.json

```diff
@@ -8,17 +8,20 @@
           "group_type": "Group type"
         }
       },
-      "cover": {
+      "binary_sensor": {
         "description": "Select group options",
         "data": {
+          "all": "All",
+          "device_class": "Group device class",
           "entities": "Group members",
           "name": "Group name"
```

Comment:
I don't think we need to prefix this all with "Group". We're creating/updating a group, what else would it be ?

Reply:
Fixed 👍